docs/two-phase-commit-transactions.md

```diff
@@ -5,6 +5,21 @@ With Two-phase Commit Transactions, you can execute a transaction that spans mul
 
 This document briefly explains how to execute Two-phase Commit Transactions in ScalarDB.
 
+## Overview
+
+ScalarDB transactions normally execute in a single transaction manager instance.
```

Comment:
I applied the suggestion in 0db8458fcbd8fe14999231b2da301bff4fd0602b.

> Also, it might be helpful to have a diagram like this.
> https://speakerdeck.com/scalar/scalar-db-universal-transaction-manager?slide=27

Regarding this, let's do in another PR.